rudder-web/src/main/scala/com/normation/rudder/web/rest/RestExtractorService.scala

```diff
@@ -624,10 +624,28 @@ case class RestExtractorService (
    * ] }
    */
   def extractNodePropertiesrFromJSON (json : JValue) : Box[RestNode] = {
-    import net.liftweb.json.JsonParser._
-    implicit val formats = DefaultFormats
+    import com.normation.utils.Control.sequence
 
-    Box(json.extractOpt[RestNode]) ?~! "Error when extracting node information"
+    for {
+      props <- json \ "properties" match {
+        case JArray(props) => Full(props)
+        case x             => Failure(s"""Error: the given parameter is not a JSON object with a 'properties' key""")
+      }
+      seq   <- sequence(props) { p =>
+                 p match {
+                   case JObject(JField("name", JString(nameValue)):: JField("value", valueValue) :: Nil) =>
```

Comment:
You're right, it would impose to use a value "", and I'm not sure it will work. I'm updating that.